### PR TITLE
Added Readme FAQ on how to set up SEO using react-snap and react-helmet

### PR DESCRIPTION
**Pull request by ruggerocastagnola**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -560,9 +615,147 @@ module.exports = buildWebpackConfig([
...
+
+As specified in the [`react-helmet` documentation](https://github.com/nfl/react-helmet). E.g.
+
+```
+export default class Home extends React.Component {
+  render() {
+    return (
+      <Helmet>
+        <title>Home title</title>
+        <meta name="robots" content="index,follow" />
```
Comment (stereobooster) [inline README.md]: if you want SEO, why do you prohibit indexing?

File: README.md [hunk 2]
```diff
@@ -560,9 +615,147 @@ module.exports = buildWebpackConfig([
...
+On 404 pages, `react-snap` requires you to have the string `404` to be part of the `<title>`, such as
+
+```
+<title>404 - Page not found</title>
+```
+
+or the following meta to be added to the `<Helmet>` component:
+
+```
+<meta name="prerender-status-code" content="404" />
```
Comment (stereobooster) [inline README.md]: this is not implemented

File: README.md (hunk 1 above)
Comment (ruggerocastagnola) [reply README.md]: Hi stereobooster, thank you for taking the time to make comments on this pr, very appreciated, and thanks for the amazing work with react-snap. I agree this is redundant and will be removed.

File: README.md (hunk 2 above)
Comment (ruggerocastagnola) [reply README.md]: Good catch, I'll reference [#91](https://github.com/stereobooster/react-snap/issues/91) instead

**Pull request closed without merging**